styles/components/Map-utils.scss

```diff
@@ -0,0 +1,20 @@
+.map-control {
```

Comment:
let's name this file
MapUtil.scss
and use it as name of component

.MapUtil {}
.MapUtil-btn {}